library/src/main/java/com/novoda/storagepathfinder/ExternalFileDirectoryInspector.java

```diff
@@ -14,8 +14,14 @@
  */
 public class ExternalFileDirectoryInspector implements SecondaryDeviceStorageInspector {
 
+    private static final StoragePath.Type SECONDARY = StoragePath.Type.SECONDARY;
     private static final String ANDROID_PATTERN = "/(Android)";
 
+    public enum Filter {   // TODO kill this soon
```

Comment:
I'd like to refactor this class so we can make better use of the repeated logic, and not have to pass down this enum through multiple methods (that don't all use it)